koschei/frontend/views.py

```diff
@@ -312,6 +317,23 @@ def to_ts(col):
             .limit(builds_per_page)\
             .all()
 
+        # prepare boundary build - the build when state changed from ok to failing
+        if builds and builds[0].state == Build.FAILED:
```

Comment:
That won't work if latest build (build[0]) is running, but pacakge status is failing. In this case we still want to boundary_build